google/cloud/spanner_dbapi/connection.py

```diff
@@ -34,7 +34,9 @@
 from google.rpc.code_pb2 import ABORTED
 
 
-AUTOCOMMIT_MODE_WARNING = "This method is non-operational in autocommit mode"
+TRANSACTION_NOT_BEGUN_WARNING = (
```

Comment:
nit: Can we change 'begun' to 'started' (both in the constant name and in the actual text)

Reply:
Changed to CLIENT_TRANSACTION_NOT_STARTED_WARNING based on our offline discussion